src/components/catalogue/AutoCompleteComponent.svelte

```diff
@@ -239,7 +237,7 @@
     /**
      * watches the input value and updates the input options
      */
-    run(() => {
```

Comment:
Use `$derived.by` here instead of `$effect`: https://svelte.dev/docs/svelte/$effect#When-not-to-use-$effect

I think in all the other places we are writing to stores so we need to use `$effect`.